packages/engine/src/camera/systems/CameraSystem.ts

```diff
@@ -224,8 +221,8 @@ export class CameraSystem extends System {
       }
       addComponent(Engine.activeCameraEntity, DesiredTransformComponent, {
         lockRotationAxis: [false, true, false],
-        rotationRate: isMobile ? 5 : 3.5,
-        positionRate: isMobile ? 3.5 : 2
+        rotationRate: window?.innerWidth <= 768 ? 3.5 : 5,
+        positionRate: window?.innerWidth <= 768 ? 2 : 3.5
```

Comment:
I think you incorrectly flipped the multipliers here; but also, we should be sure we understand the rationale for these multipliers; what is the basis for the different multipliers? Is it because of screen size, or it is because of differences in touch vs mouse input? This ambiguity (with “isMobile”) is one of the things we should really try to resolve here.